### DISCUSSION TITLE
Sunbird ED Showcase on 6th April

### DISCUSSION THREAD

**Discussion by chitranshu-keshav** [Sunbird-Ed/Community#453] [General Discussion]
There will be a weekly Sunbird ED showcase every Friday from 11:30 AM to 12:30 PM. The community members can utilize this showcase to demonstrate the functionalities/changes they are building in the upcoming releases. This will help to keep the community updated on the upcoming changes as well as help in getting early feedback on the same

Since coming Friday (7th April) is a holiday, this weeks showcase is planned on Thursday (6th April) between 11:30 AM and 12:30 PM. Below is the zoom link for the event. Please reply back to this thread with the item that you would like to demonstrate on 6th April showcase

Topic: Sunbird ED Weekly demo
Time: Apr 6, 2023 11:30 AM India

**[Join Zoom Meeting Link](https://us06web.zoom.us/j/81195665168?pwd=bkxKQ1E2UzFqZHRUVEkzUGdOWEV6Zz09)**

Meeting ID: 811 9566 5168
Passcode: 671247

@vinukumar-vs @parthlawate @Prateek-slokam @kiranharidas187 @Krishnaj20 @rajeevsathish @swayangjit @gandham-santhosh 
cc: @madhucr @vrayulu @SanthoshVasabhaktula

**Comment by parthlawate** [comment]
Hi, Can we add these to some event calendar so its easier to track the event ? In Sunbird RC we are tagging the event in the Discord community.

**Reply by chitranshu-keshav** [reply]
@parthlawate Sure it will be helpful for all. Do you have any suggestions how we can do it for ED?

**Reply by parthlawate** [reply]
If we have a discord channel, we can do the same as RC ?

**Reply by chitranshu-keshav** [reply]
Sunbird ED does not has a discord channel. We will have to use the existing channels i.e this forum, microsite or sunbird site for the same

**Reply by parthlawate** [reply]
OK.. then maybe create a Sunbird Events Google Calendar and embed it somewhere - the Sunbird Website or microsite page ?
